.travis.yml

```diff
@@ -7,11 +7,17 @@ go:
 
 sudo: required
 
+services:
+  - docker
+
 install:
+  - docker build -t ethereum/sharding-p2p:latest .
+  - docker run -d --name p2p ethereum/sharding-p2p:latest
```

Comment:
1. Can be replaced by `make build` and `make run`?
2. Should `docker run` be moved to `script` or other section?

Reply:
prefer being explicit here and decouple with the Makefile.

Reply:
Get it. Thanks!